docs/ProfilerOptions.md

```diff
@@ -0,0 +1,113 @@
+| `--filter FILTER`  | `filter=FILTER`   | Filter threads with thread ids during wall-clock profiling mode                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| `--fdtransfer`     | `fdtransfer`      | Runs a background process that provides access to perf_events to an unprivileged process. `--fdtransfer` is useful for profiling a process in a container (which lacks access to perf_events) from the host.</br>See [Profiling Java in a container](#https://github.com/async-profiler/async-profiler/blob/master/docs/ProfilingInContainer.md).                                                                                                                                                                                                                                                                                                                                                                                           |
+| `-v --version`     | N/A               | Prints the version of profiler library. If PID is specified, gets the version of the library loaded into the given process.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
`version` is applicable to an agent, too.

Reply:
Addressed in [e6282c8](https://github.com/async-profiler/async-profiler/pull/1029/commits/e6282c83bc0aa6007df310fd5e638e9d4d5ed6f1)